src/main/java/com/autotune/database/table/KruizeExperimentEntry.java

```diff
@@ -55,6 +55,7 @@ public class KruizeExperimentEntry {
     private String mode;
     private String target_cluster;
     private String performance_profile;
+    private String experiment_type;
```

Comment:
It should follow camelCase convention, and  should use the @Json decorator. However, I notice others using snake_case. Ideally, it should be camelCase, with the Gson decorator used to rename the JSON attributes.

Reply:
Using camelCase with experiment entries can cause readability issues, as SQL is case-insensitive, and it would become experimententry in queries, which isn't very readable. I think that's why snake_case is being used here. I haven't updated it to camelCase yet. What do you think—should I go ahead and make the change?

Reply:
ok seems fine